### PR TITLE
Adds grey speech problem(wingdings)

### PR DESCRIPTION
**Pull request by SplinterGP**
adds a speech problem that causes grey speak to be completely messed on wingdings(-and lets legion bully the twink with it for requesting spice-), also adds the psionic communication color that dind't had on CSS.

### CODE REVIEW

File: code/modules/client/preference_setup/general/03_body.dm [hunk 1]
```diff
@@ -807,6 +814,10 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 		pref.bgstate = next_in_list(pref.bgstate, pref.bgstate_options)
 		return TOPIC_REFRESH_UPDATE_PREVIEW
 
+	else if(href_list["wingdings"])
+		pref.wingdings = !pref.wingdings
+		return TOPIC_REFRESH_UPDATE_PREVIEW
+
```
Comment (Repede) [inline code/modules/client/preference_setup/general/03_body.dm]: Missing yw comments

File: code/modules/client/preference_setup/general/03_body.dm [hunk 2]
```diff
@@ -125,6 +127,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 	character.g_synth	= pref.g_synth
 	character.b_synth	= pref.b_synth
 	character.synth_markings = pref.synth_markings
+	character.wingdings = pref.wingdings
```
Comment (Repede) [inline code/modules/client/preference_setup/general/03_body.dm]: missing yw comment

File: code/modules/client/preference_setup/general/03_body.dm [hunk 3]
```diff
@@ -70,6 +71,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 	S["synth_markings"]		<< pref.synth_markings
 	S["bgstate"]			<< pref.bgstate
 	S["body_descriptors"]	<< pref.body_descriptors
+	S["Wingdings"]          << pref.wingdings
```
Comment (Repede) [inline code/modules/client/preference_setup/general/03_body.dm]: missing yw comment

File: code/modules/client/preference_setup/general/03_body.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/client/preference_setup/general/03_body.dm]: Added.

**Comment by SplinterGP** [comment]
Funnily that extra <b> would compile just fine, so i don't see it, thankfully we have Travis.

**Pull request merged**